packages/styles/src/layouts/portal/_type.scss

```diff
@@ -53,6 +53,9 @@
   h5,
   .h5 {
     margin-top: map.get(spacing.$post-sizes, 'bigger-big');
+    &:first-child {
+      margin-top: 0;
+    }
```

Comment:
This selector is too generic, scope it to the notification overlay title, no matter what heading level.